airbyte-webapp/nginx/default.conf.template

```diff
@@ -6,6 +6,8 @@ server {
     #charset koi8-r;
     #access_log  /var/log/nginx/host.access.log  main;
 
+    resolver_timeout 30s;
```

Comment:
Increasing this since we need to wait for the server to become available; otherwise the proxy_pass fails.